qutip/core/cy/coefficient.pyx

```diff
@@ -253,6 +283,15 @@ cdef class StepCoefficient(Coefficient):
     def __reduce__(self):
         return (StepCoefficient, (self.coeff_np, self.tlist_np, self.cte))
 
+    cpdef Coefficient copy(self):
+        return StepCoefficient(self.coeff_np, self.tlist_np, self.cte)
+
+    def replace(self, *, arguments=None, tlist=None):
+        if tlist:
+            return InterCoefficient(self.coeff_np, tlist)
+        else:
+            return self.copy()
+
```

Comment:
Is `InterCoefficient` a mistake here, or is there a reason for the replacement?

Reply:
It is a mistake.